tests/Unit/DataStructures/Test_Mesh.cpp

```diff
@@ -112,6 +118,23 @@ SPECTRE_TEST_CASE("Unit.DataStructures.Mesh", "[DataStructures][Unit]") {
         mesh3d_sliced_2, {{2, 3}},
         {{Spectral::Basis::Legendre, Spectral::Basis::Legendre}},
         {{Spectral::Quadrature::GaussLobatto, Spectral::Quadrature::Gauss}});
+    CHECK(mesh3d.slice_through() == Mesh<0>{});
+    CHECK(mesh3d.slice_through(0, 1) == mesh3d_sliced_2);
+    CHECK(mesh3d.slice_through(0, 2) == mesh3d_sliced_1);
+    CHECK(mesh3d.slice_through(1, 2) == mesh3d_sliced_0);
```

Comment:
What happens if I request `(2, 1)` or `(2, 0, 1)`? This should be documented and tested

Reply:
Or `(0, 0, 0)` for that matter.  (That one probably should not work.)